account/manager.go

```diff
@@ -624,9 +691,18 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 				}
 			}
 
-			acctKey := account.TraderKey.PubKey.SerializeCompressed()
-			contextLabel := fmt.Sprintf(" poold -- "+
-				"AccountCreation(acct_key=%x)", acctKey)
+			txFee, err := m.deriveFeeFromTx(ctx, accountTx)
+			if err != nil {
```

Comment:
Since this depends on the TX fee being calculated via a call to `lnd`, we shouldn't abort/return here in case of an error. I guess just logging and setting the fee to zero is sufficient.

Reply:
I'm concerned that a user might not notice that an error has occurred if the fee is a valid value (0). Is a log message really sufficient? For example, the sum total fee might still seem reasonable.

But I'm not sure what we can do otherwise! What are your thoughts please?

Reply:
I suppose ideally we would set the fee to json `null`. Which I think means using a `*btcutil.Amount` here: https://github.com/lightninglabs/pool/pull/396/files#diff-94465ef38c605b8bddd6e99ee32de13f57b8d6135d6be738f750498e82f19c8eR132

Do you agree?

Reply:
Given that a fee of 0 sats is an invalid value for any on-chain transaction, I think it's enough to not break the account funding flow because of a call to lnd that is only for statistics reason. And given that we are going to migrate this to PSBT funding which gives us way more info anyway, I think that's okay for now.

Reply:
I'll go with your suggestion and use 0 fee.

> I think it's enough to not break the account funding flow because of a call to lnd that is only for statistics reason

I'm not sure I completely understand what you mean. Wouldn't the use of `*btcutil.Amount` would be isolated to labels? Shouldn't touch the rest of the account funding flow.

Reply:
Ah, my comment was more about whether or not we should return an error or just log something.

Okay, I think I know what you mean now why `nil` is better than 0. That will cause any calculation in the UI to fail (or require special handling) instead of just silently summing up with 0. So yeah, maybe we should actually do that. Sorry for the back and forth, context switching a bit too much at the moment.